scripts/deploy.sh

```diff
@@ -50,14 +50,23 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   fi
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
-  if [[ ! -d /etc/openstack_deploy/ ]]; then
+  if [[ ! -e /openstack ]]; then
```

Comment:
I changed to checking /openstack because I wanted to be able to create /etc/openstack_deploy/openstack_user_config.yaml before running deploy.sh, and not have it overridden. Will update the commit message.